README.md

```diff
@@ -41,13 +41,32 @@ Formats the token amount.
 #### Parameters
 
 - `options.symbol`: the token symbol, as a `String`. Overrides the value set in the constructor.
+- `options.commify`: whether the token amount should have comma separators
 - `options.digits`: the number of digits to display. Defaults to `2`.
 - `options.displaySign`: whether the sign (`-` or `+`) should be displayed for the amount.
 
 ### TokenAmount#format(options)
 
 Alias to TokenAmount#format().
 
+### TokenAmount#convert(rate, decimals, options)
+
+Converts from a rate, returning a new `TokenAmount` instance with the desired decimals and set
```

Comment:
We should write a bit more about the type of rate we expect! This is always, always super confusing (which side of the exchange are we on? Is it in the same units, or basis points, or etc?)

Reply:
Yeah! Completely agree! In https://github.com/aragon/token-amount/pull/7/commits/b4344ce174b62be5994de3fdce8cf18642ba0267 I added some JSDoc to explain how it works; will put it here as well and expand it a bit more.

Reply:
Done in https://github.com/aragon/token-amount/pull/7/commits/f249e00b4ebc52945bf6c24b79b41d86124255f6